docs/source/ko/_toctree.yml

```diff
@@ -0,0 +1,74 @@
+- title: "시작하기"
+  sections:
+    - local: index
+      title: 홈
+    - local: quick-start
+      title: 둘러보기
+    - local: installation
+      title: 설치 방법
+- title: "(번역 중) How-to 가이드"
+  sections:
+    - local: guides/overview
+      title: 개요
+    - local: guides/download
+      title: 파일 다운로드하기
+    - local: in_translation
+      title: (번역 중) Upload files
```

Comment:
Hi @Wauplin, thank you for your helpful feedback!

There was a mistake in deleting `overview.md` instead of `upload.md`. It has been fixed in a new commit.
The document will be proof-read for the next 3 days. Please feel free to share any suggestions or comments. Otherwise, I think we can merge it after that.